packages/core/src/utilities/line-height/_line-height.docs.scss

```diff
@@ -3,17 +3,9 @@ Line height
 
 In CSS, the `line-height` property controls the leading (`led-ing`). Leading refers to the vertical distance from the baseline of one line to the baseline of the next.
 
-**Format**: `ds-u-leading--[NAME]`
-
-Markup:
-<h2 class="preview__label">ds-u-leading--lead</h2>
-<div class="ds-u-leading--lead">{{lorem-l}}</div>
-<h2 class="preview__label">ds-u-leading--base</h2>
-<div class="ds-u-leading--base ds-u-font-size--base">{{lorem-l}}</div>
-<h2 class="preview__label">ds-u-leading--heading</h2>
-<div class="ds-u-leading--heading">{{lorem-l}}</div>
-<h2 class="preview__label">ds-u-leading--reset</h2>
-<div class="ds-u-leading--reset">{{lorem-l}}</div>
+**Format**: `ds-u-leading--[lead|base|heading|reset]`
```

Comment:
I think it would be helpful to show the spacing value for each leading, for example, base is `1.5`